package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "seed": "node db/faker.js",
-    "server": "nodemon server -i client",
+    "server": "nodemon server/server.js -i client",
```

Comment:
What is '-i client' meant to be doing here?

Reply:
it ignores the client so it doesn't keep refreshing the server when I change stuff on the client

Reply:
Great. I'll be doing the same.
Thank you.